ext/standard/tests/math/is_finite_basic.phpt

```diff
@@ -2,26 +2,29 @@
 Test is_finite() - basic function test is_finite()
 --FILE--
 <?php
-$values = array(234,
-                -234,
-                23.45e1,
-                -23.45e1,
-                0xEA,
-                0352,
-                "234",
-                "234.5",
-                "23.45e1",
-                true,
-                false,
-                pow(0, -2),
-                acos(1.01));
-;
+$values = [
+    234,
+    -234,
+    23.45e1,
+    -23.45e1,
+    0xea,
+    0352,
+    "234",
+    "234.5",
+    "23.45e1",
+    true,
+    false,
+    pow(0, -2),
+    acos(1.01),
+];
```

Comment:
That's true. I reformatted the code as I have already touched the file within PR.

Reply:
Unrelated changes, such as this reformatting, make it harder to see what changed in the test over time, e.g. within the `git blame` output: As a reader I would need to think whether the changes are significant or not. As an example, this change makes it harder to determine if this RFC added a new test case to properly cover the changed code.

Reply:
Do you think it would be good to reformat all such tests in a separate PR? I also had some cases in the past when I was modifying existing tests and the code was not properly formatted so I just included this in PR. If we reformat everything at once then we don't have such discussions again.

Reply:
I'm generally against reformatting, but this formatting is just objectively bad. So, doing it in a separate PR makes sense to me.